### PR TITLE
Cockpit 228

### PR DESCRIPTION
**Pull request by KKoukiou**
Preview https://kkoukiou.github.io/cockpit-project.github.io/blog/cockpit-228.html

### CODE REVIEW

File: _posts/2020-09-16-cockpit-228.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+summary: Cockpit
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly. Here are the release notes from Cockpit version 228.
+
+### Accounts: Allow setting up weak passwords
+
+Cockpit now allows users to set up a weak password. The user gets notified that
+the password password is weak, but clicking the submit button again will use the
```
Comment (marusak) [inline _posts/2020-09-16-cockpit-228.md:17]: password password password :)

File: _posts/2020-09-16-cockpit-228.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+![Accounts weak password](/images/accounts-weak-password.png){:.screenshot}
+
+### Changes to remote host logins
+
+Cockpit now never tries to log into remote hosts with your initial login password,
+and consequently does not promote password reuse across accounts anymore.
+Instead, Cockpit now helps with setting up SSH keys for users that want automatic
+and password-less login to remote hosts.
+
+![Remote host logins](/images/remote-hosts-logins.png){:.screenshot}
```
Comment (martinpitt) [inline _posts/2020-09-16-cockpit-228.md:29]: The [last demo video](https://www.youtube.com/watch?v=4UBZWKk-T28) is reasonably close to the current experience, just a tad sloppy at the beginning. We could link to it saying that it's a demo, just perhaps not show it inline? It gives a much better demonstration what this is all about than a simple screenshot.

File: _posts/2020-09-16-cockpit-228.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+### Machines: Add support for reverting and deleting VM snapshots
```
Comment (martinpitt) [inline _posts/2020-09-16-cockpit-228.md:31]: "revert a snapshot" sounds wrong -- we don't "undo the snapshot", we "undo the changes since the snapshot". I would have said "Apply a snapshot", but as the button/UI says "Revert", that's burned. But how about "revert to"?

File: _posts/2020-09-16-cockpit-228.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+One can now restore a VM guest to a snapshot that has already been created.
```
Comment (martinpitt) [inline _posts/2020-09-16-cockpit-228.md:33]: "that has already been created" is tautological IMHO. How about simply ".. to a snapshot", or "to a previously created snapshot"?

File: _posts/2020-09-16-cockpit-228.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+thus it was finally removed upstream as well.
+If you still use it on another distribution or from upstream builds, please migrate now, or build it from an older upstream release.
+
+### Try it out
+
+Cockpit 228 is available now:
+
+ * [For your Linux system](https://cockpit-project.org/running.html)
+ * [Cockpit Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/228)
+ * [Cockpit Fedora 31](FIXME)
```
Comment (martinpitt) [inline _posts/2020-09-16-cockpit-228.md:53]: 31 → 33

**Review by martinpitt — CHANGES_REQUESTED** [review]
Thanks! Some nitpicks.

File: _posts/2020-09-16-cockpit-228.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+Cockpit now allows users to set up a weak password. The user gets notified that
+the password is weak, but clicking the submit button again will use the selected password anyway.
+This is similar to how weak passwords are handled in the Anaconda installer.
+
+![Accounts weak password](/images/accounts-weak-password.png){:.screenshot}
+
+### Changes to remote host logins
+
+Cockpit now never tries to log into remote hosts with your initial login password,
```
Comment (mvollmer) [inline _posts/2020-09-16-cockpit-228.md:24]: I think it is too easy to miss the "never" here and read this is as "Cockpit now tries to log into remote hosts with your initial login password".

What about:

Cockpit used to try to log into remote hosts with your initial login password when "Reuse my password for remote connections" was selected during login. This option has been removed and consequently ...

File: _posts/2020-09-16-cockpit-228.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+and consequently does not promote password reuse across accounts anymore.
+Instead, Cockpit now helps with setting up SSH keys for users that want automatic
+and password-less login to remote hosts.
+
+Here is a quick [demo](https://www.youtube.com/watch?v=4UBZWKk-T28).
```
Comment (mvollmer) [inline _posts/2020-09-16-cockpit-228.md:29]: Let's also show the screenshot to give people a quick idea of what this is about.

**Review by mvollmer — CHANGES_REQUESTED** [review]

File: _posts/2020-09-16-cockpit-228.md (hunk 2 above)
Comment (martinpitt) [reply _posts/2020-09-16-cockpit-228.md]: I put back the screenshot, and still keep the demo video link.

File: _posts/2020-09-16-cockpit-228.md (hunk 6 above)
Comment (martinpitt) [reply _posts/2020-09-16-cockpit-228.md]: I reworked it accordingly.

**Review by martinpitt — APPROVED** [review]
LGTM now. @mvollmer, can you please cross-check the SSH bits again? @garrett, do you have some time for some language beautification? Thanks!

**Review by mvollmer — APPROVED** [review]

**Review by garrett — APPROVED** [review]
I've updated the last screenshot a bit and force pushed the changes. Everything else looks good enough to me. :wink:

**Comment by KKoukiou** [comment]
Please someone merge - I don't have permission to merge here.

**Comment by garrett** [comment]
Yeah, I think the repo is misconfigured and only @martinpitt can merge.
